app/views/AttachmentView.tsx

```diff
@@ -123,7 +123,11 @@ class AttachmentView extends React.Component<IAttachmentViewProps, IAttachmentVi
 
 		this.setState({ loading: true });
 		try {
-			const extension = image_url ? `.${mime.extension(image_type) || 'jpg'}` : `.${mime.extension(video_type) || 'mp4'}`;
+			const extension = image_url
+				? `.${mime.extension(image_type) || 'jpg'}`
+				: `.${(video_type === 'video/quicktime' && 'mov') || mime.extension(video_type) || 'mp4'}`;
```

Comment:
This solution solves for this specific video extension, but you got to check if other extensions also have this problem, or only this one iOS can't download?

Reply:
Actually, I tested for `.mp4`, `.qt`, `.mov`.

I tried the `3gp`, but this format isn't supported, anyway